pkg/filters/processtree.go

```diff
@@ -96,6 +98,27 @@ func (filter *ProcessTreeFilter) InitBPF(bpfModule *bpf.Module) error {
 		return fmt.Errorf("could not read proc dir: %v", err)
 	}
 
+	updateBPF := func(shouldBeTraced bool, pid uint64) {
+		filterVal := make([]byte, 16)
+		var equalInScopes, equalitySetInScopes uint64
+		curVal, err := processTreeBPFMap.GetValue(unsafe.Pointer(&pid))
+		if err == nil {
```

Comment:
ditto, and I guess in all other filters.

Reply:
ditto.